tsconfig.json

```diff
@@ -3,6 +3,7 @@
 	"compilerOptions": {
 		"strict": true,
 		"lib": ["ES2021", "DOM"],
+		"skipLibCheck": true,
```

Comment:
Hi Balazs, great question. I added this since I was encountering a TS error while trying to commit my work. I'm not sure if anyone else had the same issue, but I can remove it here and create a new ticket to explore the problem in detail.